src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -83,23 +112,49 @@ export default class DataTable extends Vue {
     let cellIndex = 0;
 
     Object.keys(this.data.head).forEach((column: string) => {
-      const label = this.data.head[column].label || this.data.head[column],
+      const infoPopoverId = `info-popover-${dataTableNumber}-${column}`,
+        label = this.data.head[column].label || this.data.head[column],
+        infoIcon = this.data.head[column].description ? (
+          <chi-button
+            id={infoPopoverId}
+            variant="flat"
+            type="icon"
+            alternative-text="Info icon"
+            onclick={() => {
```

Comment:
Setting onclick={...} on custom elements like `<chi-button>` is not a good practice as it will trigger the function event if the inner `<button>` is in a disabled state. Use `chiClick` event instead, please.